### ISSUE TITLE
Question about this vs. react-a11y

### ISSUE THREAD

**Issue by reintroducing** [dylanb/react-axe#2]
Hello,
Thanks for putting this out.  I imagine you're aware of [react-a11y](https://github.com/reactjs/react-a11y) and I was hoping you could discuss how react-axe is different from react-a11y and why someone would use one over the other.  

I also think this information would be fantastic to add to the Readme.

Thanks!

**Comment by dylanb** [comment]
The main difference is that react-axe tests the accessibility of the actual DOM. This is important because many accessibility issues exist at the intersection of the DOM and the CSS and unless you have a fully rendered DOM, you will get two sorts of inaccuracies:

1) False negatives because of lacking information. Example is in order to test color contrast you must know the foreground and background colors, and
2) False positives because the element being evaluated is not in its final state and the information to communicate this to the testing algorithm is not available. Example is an inert piece of code that will be augmented once it becomes active.

**Comment by reintroducing** [comment]
Thank you and thanks for adding this to the readme, much appreciated.
